### ISSUE TITLE
Python 2 compatibility

### ISSUE THREAD

**Issue by tsalo** [wiheto/fetchopenfmri#2]
The script isn't currently Python 2 compatible, I think mostly because of urllib (this SO [post](http://stackoverflow.com/questions/24652074/importerror-no-module-named-request) covers it). I've made some changes in my fork that seem to fix the issue and can open a PR if you want.

**Comment by wiheto** [comment]
Sorry! Never did testing with Python 2 (I think I wrote that I hadn't tried). 

Gladly open a PR and I'll give it a look over and then integrate it.

**Comment by tsalo** [comment]
I've opened #3 if you want to take a look. Also, the `tarfile` section was breaking for me, but I don't know if it's a version thing. Switching from `tarfile.TarFile` to `tarfile.open` fixed it for me.

**Comment by wiheto** [comment]
Sorry I've been super busy. 

I will accept it soon. I checked it quick and decided that I need to check the tarfile.open doesn't break it for me in python3 and if a tf.close() is needed to later delete files (seemed to matter more in windows). 

Hopefully today or tomorrow. It's high up on my todo list.

**Comment by wiheto** [comment]
It seems fine now. I thought zipfile and tarfile worked identically, which led to me pushing some errors when I tried to implement your changes for both zipfile and tarfile. But your changes have been merged, I changed them, and realized you were correct from the start. 

I've tested with both python 2.7 and 3.6 and everything seems fine on my end. 

Thanks for your help! If you still have any problems, let me know.
